app/views/dashboard/_eeo_data.haml

```diff
@@ -0,0 +1,6 @@
+= form_tag eeo_data_application_records_path, method: :get do
+  List EEO data between
+  = text_field_tag :eeo_start_date, '', class: 'datepicker', required: true
```

Comment:
The datepicker JS acts on the ID. If more than one datepicker has the same ID, it will act on the first element with that ID, and since there are 3 datepickers on this page, only one of them can be named start_date and end_date. The datepickers in _reports.haml are named start_date and end_date, so I cannot make the changes you suggest.